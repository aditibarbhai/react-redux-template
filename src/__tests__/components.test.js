import React from 'react';
import { shallow } from 'enzyme';
import { HelloWorld } from '../components';

describe('Components', () => {
  it('HelloWorld should render without errors', () => {
    const wrapper = shallow(<HelloWorld />);
    expect(wrapper).toMatchSnapshot();
  });
});
