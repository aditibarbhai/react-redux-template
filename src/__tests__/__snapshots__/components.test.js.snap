// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components HelloWorld should render without errors 1`] = `
<h1>
  Hello, world!
</h1>
`;
